azext_iot/digitaltwins/params.py

```diff
@@ -289,7 +289,7 @@ def load_digitaltwins_arguments(self, _):
         )
         context.argument(
             "models",
-            options_list=["--models"],
+            options_list=["--models", "--model"],
```

Comment:
nit: I personally disagree with specifying `--model` as well as `--models`, it can make parsing non-array nested JSON a bit more challenging/ugly in code.

Reply:
e.g.

```
models = [Model(model) for model in models_json]
```

VS

```
if isinstance(models_json, list):
    models = [Model(model) for model in models_json]
else
    model = Model(models_json)
```

Reply:
The digital twins code right now can handle uploading either an array of models or a single model via `--models` ...but this `--model` addition may not be intended.

Reply:
As in this params is for a different service that is not under review

Reply:
Yeah I'd be a fan of either one or the other, but probably not both